packages/services/schema/src/orchestrators.ts

```diff
@@ -318,7 +319,8 @@ async function callExternalService(
         span.setAttribute('error.message', error.message);
         span.setAttribute('error.type', error.name);
 
-        logger.info('Network error without response. (%s)', error.message);
+        logger.error('Network error without response. (%s, %s)', error.name, error.message);
```

Comment:
@dotansimha Let's prefix the string interpolations like others.

```
logger.error('Network error without response. (errorName=%s, errorMessage=%s)', error.name, error.message);
```

Reply:
sure, fixed!